xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/viewers/tags.js

```diff
@@ -169,6 +169,7 @@ viewers.Tags = Class.create({
 
     new XWiki.widgets.Suggest(form.down('input[type=text]'), {
       script: "${xwiki.getURL('Main.WebHome', 'view', 'xpage=suggest&classname=XWiki.TagClass&fieldname=tags&firCol=-&secCol=-')}&",
+      parentContainer:"tag-add-form-suggest",
```

Comment:
Deserves a comment explaining why the default value is not good enough.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2238/commits/800ad72f3ccb496ceaa30ddab845f4c8e6ae1af7